core/lib/zksync_core/src/state_keeper/io/tests/mod.rs

```diff
@@ -431,6 +434,113 @@ async fn miniblock_and_l1_batch_processing_with_sync_sealer() {
     test_miniblock_and_l1_batch_processing(pool, 0).await;
 }
 
+// FIXME (PLA-589): Doesn't work because of missing system contracts, which cannot be added w/o a miniblock
```

Comment:
Until foreign key constraints referencing miniblocks / L1 batches are removed, it seems impossible to meaningfully test state keeper I/O implementations. @tomg10 maybe it makes sense to extract the removal of these constraints into a separate PR?